shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
Indeed, with this patch as it currently stands, the topbar gets displayed on top of the login provider modals. It doesn't *look* that bad, but when you click the topbar the modal is not dismissed.